File: MLR-spring/src/main/java/com/mlr/dto/LeakageAlertDto.java
package com.mlr.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import java.time.LocalDateTime;

@Data
public class LeakageAlertDto {
    private Long id;
    private String ruleName;
    private String message;
    private String severity; 


    @JsonProperty("isRead")
    private boolean isRead;

    private LocalDateTime createdAt;
}

File: MLR/DTOs/LeakageAlertDto.cs
namespace MLR.DTOs;

public class LeakageAlertDto
{
    public int Id { get; set; }
    public string RuleName { get; set; } = string.Empty;
    public string Message { get; set; } = string.Empty;
    public string Severity { get; set; } = string.Empty;
    public bool IsRead { get; set; } // Added property
    public DateTime CreatedAt { get; set; }
}